common/src/KokkosFFT_normalization.hpp

```diff
@@ -62,6 +62,11 @@ template <typename ExecutionSpace, typename ViewType>
 void normalize(const ExecutionSpace& exec_space, ViewType& inout,
                Direction direction, Normalization normalization,
                std::size_t fft_size) {
+  static_assert(KokkosFFT::Impl::is_operatable_view_v<ExecutionSpace, ViewType>,
+                "normalize: View value type must be float, double, "
+                "Kokkos::Complex<float>, or Kokkos::Complex<double>. "
+                "Layout must be either LayoutLeft or LayoutRight. "
+                "ExecutionSpace must be able to access data in ViewType");
```

Comment:
Is it better to merge all the checks in one `static_assert` or split them in multiple ones ?

Reply:
That may be changed in the future. For the moment, I would like to merge all checks in order not to forget some of them.